selfservice/hook/web_hook.go

```diff
@@ -209,7 +210,15 @@ func (e *WebHook) ExecutePostRegistrationPostPersistHook(_ http.ResponseWriter,
 	if gjson.GetBytes(e.conf, "can_interrupt").Bool() || gjson.GetBytes(e.conf, "response.parse").Bool() {
```

Comment:
@zepatrik PostRegistrationPostPersist hooks are always non-interrupting and the response is not parsed. So I still think it's fine to run these in a new context, as the rest of the flow is untouched by the result of this.